packages/monaco/src/typings/monaco/index.d.ts

```diff
@@ -1119,6 +1119,8 @@ declare module monaco.contextKeyService {
 
 declare module monaco.contextkey {
     export class ContextKeyExpr {
+        expr?: ContextKeyExpr[];
+        key?: string;
         static deserialize(when: string): ContextKeyExpr;
```

Comment:
expression has already `keys` property: https://github.com/TypeFox/vscode/blob/70b8db24a37fafc77247de7f7cb5bb0195120ed0/src/vs/platform/contextkey/common/contextkey.ts#L161

We should expose and use it instead.

Reply:
fixed